src/objclass/objclass.h

```diff
@@ -162,6 +162,4 @@ bool cls_has_chunk(cls_method_context_t hctx, std::string fp_oid);
 extern uint64_t cls_get_osd_min_alloc_size(cls_method_context_t hctx);
 extern uint64_t cls_get_pool_stripe_width(cls_method_context_t hctx);
 
-#endif
-
-#endif
+#endif // CEPH_OBJCLASS_H
```

Comment:
but shall we guard the methods whose names do not start with "cls_cxx", as i think this is convention we need to folow to differentiate the C++ methods and C functions? where the latter can be consumed by rados obj classes written in C language.

Reply:
I'll follow the convention.